app/delivery/routes.js

```diff
@@ -1,21 +1,26 @@
 ﻿const express = require('express');
-const { getAsteroidData } = require('../usecases/getAsteroidData');
-const { nasaApiUrl, nasaApiKey } = require('../../config');
+const {getAsteroidData} = require('../usecases/getAsteroidData');
+const {nasaApiUrl, nasaApiKey} = require('../../config');
 const Exception = require("../customExceptions/Exception");
-const { calculateDateRange } = require('../utils/utils');
+const {calculateDateRange} = require('../utils/utils');
 
 const router = express.Router();
 
 router.get('/meteors', async (req, res, next) => {
     try {
-        const { start_date, end_date, sentry_objects_count, were_dangerous_meteors } = req.query;
-        const { startDate, endDate } = calculateDateRange(start_date, end_date);
+        const {start_date, end_date, sentry_objects_count, were_dangerous_meteors} = req.query;
+        const {startDate, endDate} = calculateDateRange(start_date, end_date);
 
```

Comment:
I updated the description of query parameters a little. date - date in ISO format or an array of 2 dates
If date is an array, then the zero element is the start date, the first element is the end date
If date is a single value, then the start date and end date are equal